src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -1,57 +1,82 @@
 import React, { Suspense, useEffect, useState } from 'react';
 
+import { CoSoulArt } from 'features/cosoul/art/CoSoulArt';
 import { fileToBase64 } from 'lib/base64';
 import { updateProfileBackground } from 'lib/gql/mutations';
 import { Role } from 'lib/users';
-import { transparentize } from 'polished';
 import { useQuery } from 'react-query';
 import { useNavigate } from 'react-router';
 import { useParams } from 'react-router-dom';
-import { colors } from 'stitches.config';
 
 import { ActivityList } from '../../features/activities/ActivityList';
-import { RecentActivityTitle } from '../../features/activities/RecentActivityTitle';
 import {
   FormFileUpload,
   LoadingModal,
   ProfileSkills,
   ProfileSocialIcons,
-  scrollToTop,
 } from 'components';
 import { EditProfileModal } from 'components/EditProfileModal';
 import { useImageUploader, useToast } from 'hooks';
 import { useFetchManifest } from 'hooks/legacyApi';
-import { Edit3 } from 'icons/__generated';
+import { ExternalLink, Edit3 } from 'icons/__generated';
 import { useMyProfile } from 'recoilState';
 import { EXTERNAL_URL_WHY_COORDINAPE_IN_CIRCLE, paths } from 'routes/paths';
 import { Avatar, Box, Button, Flex, Link, MarkdownPreview, Text } from 'ui';
-import { SingleColumnLayout } from 'ui/layouts';
 import { getAvatarPath } from 'utils/domain';
 
-import { queryProfile } from './queries';
+import {
+  queryProfilePgive,
+  queryProfile,
+  QUERY_KEY_PROFILE_TOTAL_PGIVE,
+  QueryProfilePgive,
+} from './queries';
 
 import type { IMyProfile, IProfile } from 'types';
 
+const COSOUL_ART_WIDTH = '300px';
```

Comment:
No need to create these constants (and only global variable should be all caps anyway).  Just use these values in place since they are only being used once.